### PR TITLE
Payara 5.21.2 Release notes + TCK Results

### PR DESCRIPTION
**Pull request by AlanRoth**

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -283,6 +285,8 @@
 * xref:schemas/README.adoc[Schemas]
 
 * xref:jakartaee-certification/README.adoc[Jakarta EE Platform 8 certification summaries]
+** xref:jakartaee-certification/5.21.2/README.adoc[Payara Server 5.21.2]
+*** xref:jakartae-certification/5.21.2/tck-results-full-5.21.2.adoc[5.21.2 Full Profile]
```
Comment (OndroMih) [inline docs/modules/ROOT/nav.adoc:289]:
```suggestion
*** xref:jakartaee-certification/5.21.2/tck-results-full-5.21.2.adoc[5.21.2 Full Profile]
```

File: docs/modules/ROOT/pages/jakartaee-certification/5.21.2/tck-results-full-5.21.2.adoc [hunk 2]
```diff
@@ -0,0 +1,738 @@
...
++
+https://jakarta.ee/specifications/platform/8/[Jakarta EE Platform, Full Profile 8]
+- TCK Version, digital SHA-256 fingerprint and download URL:
++
+http://download.eclipse.org/jakartaee/platform/8/eclipse-jakartaeetck-8.0.2.zip[Jakarta EE Platform CTS 8.0.2]
+, SHA-256: `14a21b617bb646055c2952f1422ec04a71389fb37301e1c2969f6c3700aee965`
+
+- Public URL of TCK Results Summary:
++
+http://docs.payara.fish/v/master/jakartaee-certification/5.21.2/tck-results-full-5.21.2.html
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/jakartaee-certification/5.21.2/tck-results-full-5.21.2.adoc:24]: The URL should be now like this:
```suggestion
https://docs.payara.fish/enterprise/docs/5.21.0/jakartaee-certification/5.21.2/tck-results-full-5.21.2.html
```
We have a redirect for old gitbook URLs for older versions but new versions should use the new Antora URLs. And the number after `docs/` is correctly 5.21.0. It's the version number of the documentation. We release new docs only for new minor Payara releases, not for patch releases. Patch releases only modify the docs for the previous minor version. I know, I know, it's not easy to get this right :-/

**Review by OndroMih — CHANGES_REQUESTED** [review]
The file https://github.com/payara/Payara-Enterprise-Documentation/blob/master/docs/modules/ROOT/pages/jakartaee-certification/README.adoc also needs to be modified to include version 5.21.2 and link to /docs/modules/ROOT/pages/jakartaee-certification/5.21.2/README.adoc

File: docs/modules/ROOT/pages/release-notes/release-notes-21-2.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
+== Supported APIs and Applications
+
+* Java EE 8 Applications
+* MicroProfile 3.3
+* Jakarta EE 8 Applications
+
+*Notes*
+
+In this release of Payara Enterprise we're unfortunately temporarily reverting an improvement we added in 5.21 - HTTP/2 support for JDK Native ALPN APIs (FISH-31). It has proved to be unstable in certain situations and so we've taken the decision to remove it from the Enterprise distributions until it's matured further - we were too quick to bring this feature in to Enterprise and apologise if this impacts you. This reversion means that HTTP/2 support will not work when using JDK 8u252 or higher (including JDK 11), and a warning will be printed in the log related to this. HTTP/2 should still work when using an older supported version of JDK 8 (8u162-8u242).
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/release-notes/release-notes-21-2.adoc:9]: official number is 5.21.0 (and not 5.21)

File: docs/modules/ROOT/pages/release-notes/release-notes-21-2.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+*Bug Fixes*
+
+* [https://github.com/payara/Payara-Enterprise/pull/171[_FISH-37_]] - @DataSourceDefinition passes serverName and URL DataSource in some cases
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/release-notes/release-notes-21-2.adoc:13]: Other release notes don't have Jira number in Italic. Can be fixed in another PR.

**Review by rdebusscher — APPROVED** [review]

**Pull request merged**